src/policy/fees.h

```diff
@@ -138,9 +138,9 @@ class CBlockPolicyEstimator
 
     /** Decay of .962 is a half-life of 18 blocks or about 3 hours */
     static constexpr double SHORT_DECAY = .962;
-    /** Decay of .998 is a half-life of 144 blocks or about 1 day */
+    /** Decay of .9952 is a half-life of 144 blocks or about 1 day */
```

Comment:
Any idea why these numbers changed but not the associated time period?

Reply:
Yeah, the comment was wrong and I think morcos fixed the value but forgot about the comment. The comment change happened in this commit https://github.com/bitcoin/bitcoin/commit/b649e0395464a659f4b3485ec71d28dc95ba48bd , and the variable it refers to no longer exists.

Reply:
Oh I see, the bucket size was changed, thanks.